pkgs/data/fonts/cozette/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, mkfontscale }:
+
+stdenv.mkDerivation rec {
+  pname = "Cozette";
+  version = "1.5.1";
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/slavfox/Cozette/releases/download/v." + version
+        + "/cozette.otb";
+      sha256 = "05k45n7jar11gnng2awpmc7zk9jdlzd6wz87xx49cp75jm4z9xm8";
+    })
+
+    (fetchurl {
+      url = "https://github.com/slavfox/Cozette/releases/download/v." + version
+        + "/CozetteVector.otf";
+      sha256 = "1sqhnjpizn1wi26lc7z2zml7yr7zkcpa72mh1drvd74rlcs1ip30";
+    })
+
+    (fetchurl {
+      url = "https://github.com/slavfox/Cozette/releases/download/v." + version
+        + "/CozetteVector.ttf";
+      sha256 = "1q4ml8shv9lmyc6bwhffwvbvl92s73j7xkb0rkqvci4f0zbz7mcy";
+    })
+  ];
+
+  nativeBuildInputs = [ mkfontscale ];
+
+  sourceRoot = "./";
+
+  unpackCmd = ''
+    otName=$(basename $(stripHash $curSrc))
+    cp $curSrc ./$otName
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/misc
+    mkdir -p $out/share/fonts/truetype
+    cp -a *.otf $out/share/fonts/misc/
```

Comment:
I believe `.otf` fonts should go to `$out/share/fonts/opentype` -- this is why I think that: https://github.com/NixOS/nixpkgs/blob/92442113a39bd1c4d6b9923df71c95a70e32de8f/pkgs/data/fonts/unscii/default.nix#L32

Reply:
Okay.  Done :)